docs/reference/tctl/activity.md

```diff
@@ -7,19 +7,11 @@ tags:
   - tctl
 ---
 
-<!-- prettier-ignore -->
-import * as WhatIsAnActivityExecution from '/docs/content/what-is-an-activity-execution.md'
+The `tctl activity` commands operate [Activity Executions](/docs/content/what-is-an-activity-execution).
```

Comment:
I get it, but somehow seems off -
Maybe something like this feels a bit more on point? WDYT?
```suggestion
The `tctl activity` command enables [Activity Execution](/docs/content/what-is-an-activity-execution) operations.
```

Reply:
Yes, that's definitely closer. (I switched the numbers back, though, because of multiple `tctl activity` commands.)